infrastructure/environments/build.tfvars

```diff
@@ -0,0 +1,308 @@
+    }
+  },
+  {
+    name = "nsip-representation"
+    subscriptions = {
+      "nsip-representation" = {},
+    }
+  },
+  {
+    name = "nsip-s51-advice"
+    subscriptions = {
+      "nsip-s51-advice" = {},
+    }
+  },
+]
+
+spark_pool_enabled         = true
+spark_pool_max_node_count  = 12
+spark_pool_min_node_count  = 3
+spark_pool_node_size       = "Small"
+spark_pool_timeout_minutes = 60
+spark_pool_version         = "3.4"
+new_spark_pool_version     = "3.4"
+
+spark_pool_preview_enabled = true
+spark_pool_preview_version = "3.4"
+
+sql_pool_collation = "SQL_Latin1_General_CP1_CI_AS"
+sql_pool_enabled   = false
+sql_pool_sku_name  = "DW100c"
+
+sql_server_administrator_username = "sqladmin"
+sql_server_enabled                = false
+
+synapse_aad_administrator = {
+  username  = "pins-odw-data-preprod-syn-ws-sqladmins"
+  object_id = "ba5af92f-a1bf-4332-a3c9-613a0a8f1b12"
+}
+
+synapse_data_exfiltration_enabled  = false
+synapse_sql_administrator_username = "synadmin"
+
+synapse_role_assignments = [
+  { # pins-odw-data-preprod-syn-ws-administrators
+    role_definition_name = "Synapse Administrator",
+    principal_id         = "be52cb0c-858f-4698-8c40-3a5ec793a2e3"
+  },
+  # This one seems to be automatically assigned when the Synapse workspace is created
+  #{ # Azure DevOps Pipelines - ODW Build - Infrastructure
+  #  role_definition_name = "Synapse Administrator",
+  #  principal_id         = "9d7c0f07-9839-4928-8927-bfc19f9f6bd2"
+  #},
+  { # pins-odw-data-preprod-syn-ws-contributors
+    role_definition_name = "Synapse Contributor",
+    principal_id         = "d59a3e85-58db-4b70-8f88-3f4a4a82ee27"
+  },
+  { # pins-odw-data-preprod-syn-ws-computeoperators
+    role_definition_name = "Synapse Compute Operator",
+    principal_id         = "f9c580cd-cab0-4c49-9f50-290194ade29e"
+  }
+]
+
+tags = {}
+
+tenant_id = "5878df98-6f88-48ab-9322-998ce557088d"
+
+vnet_base_cidr_block          = "10.100.0.0/24"
+vnet_base_cidr_block_failover = "10.100.1.0/24"
+vnet_subnets = [
+  {
+    "name" : "AzureBastionSubnet",
+    "new_bits" : 4 # /28
+    service_endpoints  = []
+    service_delegation = []
+  },
+  {
+    "name" : "FunctionAppSubnet",
+    "new_bits" : 4 # /28
+    service_endpoints = ["Microsoft.Storage", "Microsoft.KeyVault", "Microsoft.ServiceBus"]
+    service_delegation = [
+      {
+        delegation_name = "Microsoft.Web/serverFarms"
+        actions         = ["Microsoft.Network/virtualNetworks/subnets/action"]
+      }
+    ]
+  },
+  {
+    "name" : "SynapseEndpointSubnet",
+    "new_bits" : 2 # /26
+    service_endpoints  = []
+    service_delegation = []
+  },
+  {
+    "name" : "ComputeSubnet"
+    "new_bits" : 2 # /26
+    service_endpoints  = ["Microsoft.Storage", "Microsoft.KeyVault"]
+    service_delegation = []
+  },
+  {
+    "name" : "ApimSubnet",
+    "new_bits" : 2 # /26
+    service_endpoints  = []
+    service_delegation = []
+  },
+]
+
+external_resource_links_enabled = false

```

Comment:
Can you maybe add in the documentation when this should be set to False or True ?

Reply:
That's been added to https://github.com/Planning-Inspectorate/ODW-Service/blob/THEODW-1639-buildEnvironment/infrastructure/README.md